tommath.h

```diff
@@ -297,28 +297,41 @@ mp_err mp_init_u64(mp_int *a, uint64_t b) MP_WUR;
 /* get magnitude */
 uint32_t mp_get_mag32(const mp_int *a) MP_WUR;
 uint64_t mp_get_mag64(const mp_int *a) MP_WUR;
+unsigned long mp_get_magl(const mp_int *a) MP_WUR;
```

Comment:
It's clear that the magnitude is always positive, but I still don't like the inconsistency of having the function suffixed by **l** but returning an `unsigned long` ... but `mp_get_magul` doesn't look good, so probably they could be renamed like that?

```
mp_get_mag_u32()
mp_get_mag_u64()
mp_get_mag_ul()
mp_get_mag_ull()
```

Reply:
Understood:   I agree with renaming those functions. The mp_get_mag_u32/64 functions are already on "develop" so - actually - are out of scope for this PR.  Shouldn't we do this rename in a separate follow-up PR then?

Reply:
I mean ....  mp_get_mag32/64 are already on develop, their renaming doesn't belong to this PR.

Reply:
Okay, I wouldn't have a problem to put them in this PR as well, but fine by me to open another PR for that change.